### DISCUSSION TITLE
Monitor Intel feeds

### DISCUSSION THREAD

**Discussion by 3isenHeiM** [Security-Onion-Solutions/securityonion#9659] [General] [answered]
Is there a feature where SecurityOnion will alert me if it couldn't connect and updates the signatures ?
Especially for custom industry-specific feeds, it's important to know if your detection rules are still up-to-date.

Thanks !

**Comment by InfosecGoon** [comment, marked as answer]
I believe the Suricata container restarts whenever new rules are loaded -- would a Grafana alert that fires when so-suricata is more than X hours of uptime be high-fidelity enough for this?

**Reply by 3isenHeiM** [reply]
The only corner case I see here is if no updates is available, the container will fetch but not restart...

Anyway thanks for the hint it seems reasonable good enough.
